frame/dapps-staking/README.md

```diff
@@ -3,13 +3,13 @@ This document describes the interface for the pallet-dapps-staking.
 
 Table of Contents:
 1. [Terminology](#Terminology)
-1. [Types](#Types)
-1. [Events](#Events)
-1. [Errors](#Errors)
-1. [Calls](#Calls)
-1. [Storage](#Storage)
-1. [Referent implementatio](#Referent)
-1. [FAQ](#FAQ)
+2. [Types](#Types)
```

Comment:
Why keep the table of contents if there's no content anymore?

Reply:
Good catch, I will remove it.